### PR TITLE
Add admin configurable dialogs for the splash page and signup

### LINKED ISSUE #213: Splash page link on self-hosted should be configurable
Currently when you visit a Sandstorm server and are logged out or otherwise don't have permission to install apps, you see the splash page with text saying that you need to join the Sandstorm mailing list for an invite.

For self-hosted instances, this is totally wrong. The server's admin is the one that can send invites. It somehow hadn't occurred to me before, but this text actually looks pretty bad because it implies that you need permission from us for self-hosted servers.

We should make this configurable. A simple config file option specifying HTML to inject should be good, with a default along the lines of: "Contact the server admin for an invite."

### PR DESCRIPTION
**Pull request by jparyani**
Fixes #213

### CODE REVIEW

File: shell/shared/shell.js [hunk 1]
```diff
@@ -276,7 +276,12 @@ if (Meteor.isClient) {
       }
     },
 
-    dateString: function (date) { return makeDateString(date); }
+    dateString: function (date) { return makeDateString(date); },
+
+    splashDialog: function() {
+      var setting = Settings.findOne("splashDialog");
+      return (setting && setting.value) || "Contact the server admin for an invite.";
```
Comment (kentonv) [inline shell/shared/shell.js]: Can you share a string constant between here and admin.js?

File: shell/shared/signup.js [hunk 2]
```diff
@@ -161,6 +161,13 @@ if (Meteor.isClient) {
       Session.set("inviteMessage", undefined);
     },
   });
+
+  Template.signup.helpers({
+    signupDialog: function () {
+      var setting = Settings.findOne("signupDialog");
+      return (setting && setting.value) || "You've been invited to join this Sandstorm server!";
```
Comment (kentonv) [inline shell/shared/signup.js]: Also here.

**Comment by kentonv** [comment]
Oops, merge conflict since I just pulled the email token login change.

**Comment by jparyani** [comment]
Done.

**Pull request merged**